src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/EligibilityServiceTest.kt

```diff
@@ -216,6 +216,26 @@ class EligibilityServiceTest {
     assertThat(result).containsExactly("has no conditional release date")
   }
 
+  @Test
```

Comment:
**note:** tests to prove cases with imprisonmentStatus == "BOTUS" is not eligible for CVL licence.